parent/pom.xml

```diff
@@ -239,7 +239,7 @@
         <plugin>
           <groupId>org.codehaus.cargo</groupId>
           <artifactId>cargo-maven3-plugin</artifactId>
-          <version>1.10.9</version>
+          <version>1.10.16</version>
```

Comment:
ℹ️ I updated cargo to have access to the `wildfly35x` containerID used in `qa/integration-tests-webapps/pom.xml`